librepo/yum.c

```diff
@@ -152,9 +162,29 @@ lr_yum_repomd_record_enabled(LrHandle *handle, const char *type)
                 return TRUE;
             x++;
         }
+        // Substitution check
+        if (handle->yumslist) {
+            LrUrlVars* elem = handle->yumslist;
+            while (TRUE) {
+                LrVar* subs = elem->data;
+                if (!strcmp(subs->var, type)) {
```

Comment:
Maybe use g_strcmp0() as it's more bullet-prof (NULL won't cause SIGSEGV)

https://developer.gnome.org/glib/stable/glib-String-Utility-Functions.html#g-strcmp0